packages/lib-classifier/src/store/WorkflowStepStore.spec.js

```diff
@@ -108,10 +118,12 @@ describe('Model > WorkflowStepStore', function () {
       })
 
       ROOT_STORE_INSTANCE = ROOT_STORE.create({
-        workflows: {
-          active: WORKFLOW
-        }
+        projects: {},
+        subjects: {},
+        userProjectPreferences: {}
       })
+      ROOT_STORE_INSTANCE.workflows.setResource(WORKFLOW)
+      ROOT_STORE_INSTANCE.workflows.setActive(WORKFLOW.id)
     })
```

Comment:
This is duplicated from line 47 so could maybe be moved into a function and declared once. `ROOT_STORE_INSTANCE = getRootStore()`.